packages/react/.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["../internal-tooling/eslint/index.js"],
+  "extends": ["../../build/eslint/index.js"],
```

Comment:
Should use package instead? I.e. `@fluentui/internal-tooling`?

Reply:
Probably, I'll see if it works at build time